spec/observables/from-promise-spec.js

```diff
@@ -109,24 +112,26 @@ describe('Observable.fromPromise', function () {
     });
   });
 
-  it('should globally throw unhandled errors', function (done) {
-    var invoked = false;
-    process.on('uncaughtException', function (reason, p) {
-      if (invoked) {
-        return;
-      }
-      invoked = true;
-      expect(reason).toBe('fail');
-      done();
-    });
+  if (isNode) {
```

Comment:
Changed to use simple if conditional for now. Thought about adding some function like `it.node` or `it.browser` to be executed under certain runtime, but it could be added in further depends on how many test we'll have per different runtime.